src/components/Calculator.js

```diff
@@ -0,0 +1,216 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Col, Row, Button } from 'react-bootstrap';
+
+class Calculator extends Component {
```

Comment:
- [ ] Please import `src/Logic/calculate.js `and `src/Logic/operate.js `here in `src/components/Calculator.js`.